packages/web-components/src/components/pictogram-item/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import'../pictogram-item';
+
+# Pictogram Item
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/pictogram-item)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/web-components/examples/codesandbox/components/pictogram-item)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/pictogram-item/pictogram-item';
+```
+
+or with [JSPM](https://jspm.org)
+
+```html
+<script type="module">
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/pictogram-item/pictogram-item.js';
+</script>
+```
+
+### HTML
+
+```html
+<dds-pictogram-item>
+  <svg slot="pictogram" focusable="false" preserveAspectRatio="xMidYMid meet"
+    xmlns="http://www.w3.org/2000/svg"
+    stroke="currentColor"
+    data-autoid="dds--pictogram-item__pictogram"
+    aria-label="Pictogram description" width="48" height="48"
+    viewBox="0 0 48 48" role="img" class="bx--pictogram-item__pictogram">
+    <path fill="none" stroke-linejoin="round" stroke-miterlimit="10" stroke-width=".72"
+      d="M37,32 H11c-1.1,0-2-0.9-2-2V13c0-1.1,0.9-2,2-2h26c1.1,0,2,0.9,2,2v17C39,31.1,38.1,32,37,32z M17,37h14 M24,32v5 M9,27h30">
+    </path>
+  </svg>
+  <span slot="heading">Lorem ipsum dolor sit</span>
+  <span slot="copy">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+    aliqua. Ut enim ad minim veniam
+  </span>
+  <dds-link-with-icon slot="cta">
+    Lorem ipsum dolor ${ArrowRight20({ slot: 'icon' })}
```

Comment:
@RaphaelAmadeu can you replace the `ArrowRight20` with an svg element?

Reply:
@jeffchew I've added this on my latest commit